src/components/ProjectsList/ProjectsList.tsx

```diff
@@ -23,7 +23,7 @@ export const ProjectsList = ({ projects }: IProjectsList) => {
 
   const rows = projects.map((project: Project) => [
     {
-      title: <Link to="#">{project.name}</Link>,
+      title: <Link to={project.id}>{project.name}</Link>,
```

Comment:
Let's create new component, something like `ProjectLink` with all required attributes, in this case it would be only `id: string` and then use it here